mediapipe/tasks/ios/components/containers/sources/MPPCategory.h

```diff
@@ -21,7 +21,7 @@ NS_ASSUME_NONNULL_BEGIN
  * index of the label in the corresponding label file. Typically it's used as the result of
  * classification tasks.
  */
-NS_SWIFT_NAME(ClassificationCategory)
+NS_SWIFT_NAME(ResultCategory)
```

Comment:
Just curious - Why did you decide to rename this? `

FWIW I feel that we can probably also just go with `Category`. It's really only used as a result type, and hence the user should not need to have to import it. This would allow us to maintain the MPPType -> Type mapping for our classes.

cc @khanhlvg

Reply:
Category is a reserved keyword in swift. You have to use a different name. `ResultCategory ` seemed more fitting. Hence the renaming.